pkgs/desktops/gnome-3/extensions/fuzzy-app-search/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitLab, gnome3, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-fuzzy-app-search";
+  version = "4";
+
+  src = fetchFromGitLab {
+    owner = "Czarlie";
+    repo = "gnome-fuzzy-app-search";
+    rev = "da9c15d39958d9c3b38df3b616fd40b85aed24e5";
```

Comment:
https://gitlab.com/Czarlie/gnome-fuzzy-app-search/-/issues/3

Reply:
Since this issue is not being resolved, could this be merged?

The commit I've chosen is the one that last touched the [metadata.json](https://gitlab.com/Czarlie/gnome-fuzzy-app-search/-/commit/da9c15d39958d9c3b38df3b616fd40b85aed24e5).

Reply:
Sure, thanks for the contribution.